libraries/botbuilder-dialogs-adaptive-testing/src/testRunner.ts

```diff
@@ -16,34 +16,33 @@ import { TestScript } from './testScript';
  * In charge of running Dialog Adaptive tests.
  */
 export class TestRunner {
-    private resourceExplorer: ResourceExplorer;
-    private testAdapter: TestAdapter;
-
     /**
      * Initializes a new instance of the `TestRunner` class.
      * @param resourcePath Path where the `.dialog` files are located.
      */
-    public constructor(resourcePath: string) {
+    public constructor(private resourcePath: string) {
         ComponentRegistration.add(new AdaptiveComponentRegistration());
         ComponentRegistration.add(new AdaptiveTestComponentRegistration());
         ComponentRegistration.add(new QnAMakerComponentRegistration());
         ComponentRegistration.add(new LuisComponentRegistration());
-
-        this.resourceExplorer = new ResourceExplorer();
-        this.resourceExplorer.addFolder(resourcePath, true, false);
-
-        this.testAdapter = new TestAdapter(TestAdapter.createConversation('botbuilder-dialogs-adaptive-testing'));
     }
 
     /**
      * Runs a test script with the specified name.
      * @param testName Test name.
      * @returns A Promise that represents the work queued to execute.
      */
-    public async runTestScript(testName: string): Promise<any> {
-        const script = this.resourceExplorer.loadType<TestScript>(`${testName}.test.dialog`);
+    public runTestScript(testName: string): Promise<void> {
```

Comment:
> 
```suggestion
    public async runTestScript(testName: string): Promise<void> {
```

Still need to label this `async`, right? `script.execute()` is async and the tests are using `await testRunner.runTestScript()...`

Reply:
You _only_ need `async` if you use the keyword `await` inside a function. You should almost never `return await` - see https://stackoverflow.com/questions/38708550/difference-between-return-await-promise-and-return-promise for more details.